### ISSUE TITLE
app crashing in mev_dealloc

### ISSUE THREAD

**Issue by chatterjeesubhojit** [manuelescrig/MEVFloatingButton#4]
I added the project according to provided steps but it is continuously crashing when it reaches to try section of mev_dealloc method. Can you check the same please cause even your sample app crashes over there. I tried even commenting the method call location, but still the app runs but the button doesn't appears at all.

I want to implement the example2 project in my app. Please suggest me the solution. 

Is there any step that I am missing out of it?
IS it necessary to install the pod in order to run add this in my project?

**Comment by manuelescrig** [comment]
I'm not being able to reproduce the crash. The example project version 1.2 is not crashing using Xcode 7.3.1. You should be able to play it.

Yes, in order to add this library to your project you have to install the pod as in README is explained.

**Comment by chatterjeesubhojit** [comment]
Hello,

I added the the project properly as per the steps except I have not
installed the pod yet.

The mev_willAppear method in UIScrollView+FloatingButton.m is not getting
called.
Following methods are getting called properly :

- setFloatingButtonDelegate
- setFloatingButtonView

But not any of the following are getting called:

- floatingButtonWillAppear
- floatingButtonDidAppear
- floatingButtonWillDisappear
- floatingButtonDidDisappear

Can you suggest? does this need the pod file to be installed?

--
Warm Regards,

Subhojit Chatterjee
Associate Software Engineer, Inscripts

M: +91 8691817897 | T (India): +91 22 30551000 | T (U.S.): +1 415 287 0451

Inscripts (I) Pvt. Ltd. |  An ISO 9001:2008 Certified Company
517-518 Shrikant Chambers, V.N. Purav Marg, Chembur, Mumbai – 400 071
http://www.inscripts.com

**Comment by chatterjeesubhojit** [comment]
Hello,

I added the the project properly as per the steps except I have not installed the pod yet.

The mev_willAppear method in UIScrollView+FloatingButton.m is not getting called. 
Following methods are getting called properly :

- setFloatingButtonDelegate
- setFloatingButtonView

But not any of the following are getting called:

- floatingButtonWillAppear
- floatingButtonDidAppear
- floatingButtonWillDisappear
- floatingButtonDidDisappear

Can you suggest? does this need the pod file to be installed?

**Comment by chatterjeesubhojit** [comment]
Hello,

I added the the project properly as per the steps except I have not
installed the pod yet.

The mev_willAppear method in UIScrollView+FloatingButton.m is not getting
called.
Following methods are getting called properly :
- setFloatingButtonDelegate
- setFloatingButtonView

But not any of the following are getting called:
- floatingButtonWillAppear
- floatingButtonDidAppear
- floatingButtonWillDisappear
- floatingButtonDidDisappear

Can you suggest? does this need the pod file to be installed?

**Comment by chatterjeesubhojit** [comment]
I tried to debug the case and it seems that the (Private) methods are not getting called successfully. (Public) ones are getting called on load of the viewcontroller.

**Comment by manuelescrig** [comment]
> IS it necessary to install the pod in order to run add this in my project?

Yes, in order to add this library to your project you have to install the pod as in README is explained.

**Comment by renetik** [comment]
This is crashing still

**Comment by mohamedadly** [comment]
Why is this closed?
the crash still happening.

**Comment by edalford11** [comment]
Can confirm as well this crash is still happening

**Comment by MubeenQazi** [comment]
Has any one fixed this ?
